restaurant/utils.py

```diff
@@ -94,3 +94,19 @@ def get_restaurant_list(
         restaurant_dict["latest_record"] = latest_inspection_record
         result.append(restaurant_dict)
     return result
+
+
+def get_filtered_restaurants(price, neighborhood, rating, category, page, limit):
+    filters = {}
+
+    if price:
+        filters["price__in"] = price
+    if neighborhood:
+        filters["neighborhood__in"] = neighborhood
+    if rating:
+        filters["rating__gte"] = rating
+    if category:
+        filters["category__in"] = category
+
+    filtered_restaurants = YelpRestaurantDetails.objects.filter(**filters)
```

Comment:
Maybe can include page and limit here? Since I saw them in the definition of this function.

Reply:
Done